etc/skate-style/base.mss

```diff
@@ -80,7 +80,10 @@
 /* WATER AREAS
 /* ================================================================== */
 
-Map { background-color: @water; }
+Map { 
+  background-color: @water;
+  buffer-size: 128;
```

Comment:
Setting this buffer-size [prevents cut off labels](https://app.asana.com/0/1200180014510248/1203677893563466/f). 

It seems we're doing this as part of the [generate_tiles script](https://github.com/mbta/tile-server/blob/339c8bbeced84e9e068e5ac719149f90e955168c/etc/generate_tiles.py#L97), but not explicitly setting it was causing labels to be cut off when rendering with kosmtik